<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Somma</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sottrazione</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moltiplicazione</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Divisione</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="bLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="aLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Vai!</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="rLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
